experimental/framework/frs-client/src/interfaces.ts

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IMember, IServiceAudience } from "@fluid-experimental/fluid-static";
+import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
+import { ITokenProvider } from "@fluidframework/routerlicious-driver";
+
+export interface FrsContainerConfig {
+    id: string;
+    logger?: ITelemetryBaseLogger;
+}
+
+export interface FrsConnectionConfig {
+    tenantId: string;
+    key: string;
+    orderer: string;
+    storage: string;
+    absoluteUrlBase?: string;
```

Comment:
Where is this used?

Reply:
Removed, I was playing around with the idea of letting the base hostname of the URL be configurable by the developer but I've taken this out for now